lib/verifier.rb

```diff
@@ -94,4 +94,20 @@ def verify_file filename
     verify manifest
   end
 
+  def verify_topics filename
+    manifest = Manifest.parse_file filename
+
+    print "Verify topics of " + manifest.filename + "..."    
+    topics_list =  manifest.topics
+    if topics_list.nil?
+      puts "Warning: Specify the topics"        
+    else
+      if !topics_list.kind_of?(Array) 
```

Comment:
This should already be covered by the schema validation. No need to add specific code here.